R/teal_slice.R

```diff
@@ -268,22 +280,68 @@ c.teal_slice <- function(...) {
   ans
 }
 
+#' @param x `teal_slice` object
+#' @keywords internal
+to_list <- function(x){
```

Comment:
How about registering this as an `as.list` method?

Reply:
that's pretty clever